test/e2e/hotrod-rolling-upgrade/hotrod_rolling_upgrade_test.go

```diff
@@ -71,7 +71,7 @@ func TestRollingUpgrade(t *testing.T) {
 	entriesPerCache := 100
 	spec := tutils.DefaultSpec(t, testKube, func(i *ispnv1.Infinispan) {
 		i.Spec.Replicas = int32(replicas)
-		i.Spec.Container.CPU = "1000m"
+		i.Spec.Container.CPU = "500m"
```

Comment:
I think we should remove this line entirely and depend on the `INFINISPAN_CPU` being set via the action. That way things should be faster when executing locally as we can use more resources.